k2/csrc/fsa_algo.cc

```diff
@@ -6,7 +6,9 @@
 
 #include "k2/csrc/fsa_algo.h"
 
+#include <algorithm>
```

Comment:
Your name in the copyright info is missing.

Reply:
mm..OK, thanks.